src/lib/core/components/VariableTree.tsx

```diff
@@ -169,7 +171,16 @@ export function VariableTree(props: Props) {
 
 export function VariableTreeDropdown(props: Props) {
   const { rootEntity, entityId, variableId, onChange } = props;
-  const [hideDisabledFields, setHideDisabledFields] = useState(false);
+
+  // use ShowHideVariableContext for setting initial hideDisabledFields boolean
+  const {
+    toggleShowHideVariable,
+    setToggleShowHideVariableHandler,
+  } = useContext(ShowHideVariableContext);
+  const [hideDisabledFields, setHideDisabledFields] = useState(
+    toggleShowHideVariable
+  );
```

Comment:
@dmfalke Thank you for your reviews and suggestions. 
As for this comment, setHideDisabledFields (state action) is required to make click event of toggling button so it is necessary.

Reply:
@dmfalke Ah... I got your point. So, instead of hideDisabledFields, using the context would work as well. I will check this case - there are several places to be changed then.